src/harness/sas_interface.py

```diff
@@ -404,4 +404,44 @@ def assertValidResponseFormatForApprovedGrant(self, grant_response):
       Nothing. It asserts if something about the response is broken/not per
       specs. Assumes it is dealing with an approved request.
     """
-    pass
\ No newline at end of file
+    pass
+
+  @abc.abstractmethod
+  def assertRegistered(self, registration_request,
+                       conditional_registration_data=None):
+    """Register a list of devices.
+
+    Quickly register N devices, assert registration SUCCESS, get CBSD IDs.
+    Includes injection of FCC IDs and conditional registration data.
+
+    Args:
```

Comment:
Yeah, I think we only want to use these in the case where we expect successful registration (which I think should happen in 100% of cases except REG_*). It should work well with both configurable and non-configurable tests (since we just take the registrationRequest as a parameter). I believe her reference to REG_14 was to highlight that this will work with configurable tests, not that it should necessarily be used with REG_14 specifically.